src/gluonts/nursery/duru/utils.py

```diff
@@ -0,0 +1,423 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import argparse
+import yaml
+import os
+import wandb
+import shutil
+import torch
+import numpy as np
+
+
+def str2bool(v):
```

Comment:
This looks very much like `distutils.util.strtobool`. Can't we just use that one?

Reply:
Thanks for this great point! I would like to leave this function in as there seem to be problems with boolean parsing in command line arguments that I otherwise appear. See https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse. I also tested this before.